catalog/opensearch/catalog-opensearch-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -71,6 +71,10 @@
             name="Receive Timeout" id="receiveTimeout"
             required="true" type="Integer" default="60000"/>
 
+        <AD description="Poll Interval to Check if the Source is available (in minutes - minimum 1)."
```

Comment:
It looks like connection timeout and receive timeout are supposed to be configurable and shouldn't be hardcoded.